docs/overview/about.md

```diff
@@ -8,17 +8,17 @@ description: A self-sovereign blockchain network that allows developers to expre
 
 Archway is an incentivized smart contract platform that rewards developers — ushering in next gen dApps.
 
-The protocol gives developers the tools to easily build and launch scalable cross-chain dApps, and get rewarded for the value they contribute to the network.
+The protocol gives developers the tools to quickly build and launch scalable cross-chain dApps and get rewarded for the value the dApps contribute to the network.
```

Comment:
easily is one of the verbs that we want to avoid
see https://developers.google.com/style/word-list#letter-e

> easy, easily
> What might be easy for you might not be easy for others. Try eliminating this word from the sentence because usually the same meaning can be conveyed without it.